Source/WebGPU/WebGPU/ComputePipeline.mm

```diff
@@ -188,10 +188,9 @@ static MTLSize metalSize(auto workgroupSize)
 
 void Device::createComputePipelineAsync(const WGPUComputePipelineDescriptor& descriptor, CompletionHandler<void(WGPUCreatePipelineAsyncStatus, Ref<ComputePipeline>&&, String&& message)>&& callback)
 {
-    // FIXME: Implement this
-    UNUSED_PARAM(descriptor);
-    instance().scheduleWork([strongThis = Ref { *this }, callback = WTFMove(callback)]() mutable {
-        callback(WGPUCreatePipelineAsyncStatus_InternalError, ComputePipeline::createInvalid(strongThis), { });
+    auto pipeline = createComputePipeline(descriptor);
```

Comment:
I think this is fine for now, but we should at least file a bug and add a FIXME with a link to the bug about doing this _actually_ asynchronously